spec/features/contacts_list_spec.rb

```diff
@@ -23,12 +22,26 @@
 			expect(page).to have_content("0")
 		end 
 
-		it 'displays 2 when there are 2' do 
-			@contact.appointments.create!
-			@contact.appointments.create!
-			visit "/"
-			expect(page).to have_content("2")
-		end 
+    describe 'when there are 2 appointments' do
+
+    	before(:each) do 
+    		@contact.appointments.create!
+    		@contact.appointments.create!
+    	end 
+
+			it 'displays the number 2' do 
```

Comment:
I am not sure why the indentation looks so off here, this is not how it appears in my text editor.